e2e/terraform/README.md

```diff
@@ -51,7 +51,7 @@ Linux clients or Windows clients.
 region                           = "us-east-1"
 instance_type                    = "t2.medium"
 server_count                     = "3"
-client_count_ubuntu_jammy_amd64  = "4"
+client_count_linux               = "4"
```

Comment:
This variable originally included the specific distro and architecture so that we could spin up different hosts in the same test run (although we never got around to doing so). It's not obvious to me how we're planning on feeding in the architecture/platform if not by this variable.

(Or, if we're planning on adding architecture/platform variables later, shouldn't we make the same change to the windows_2026_amd64` var?)